learning/editor/godot_3_project_settings.rst

```diff
@@ -0,0 +1,720 @@
+.. _doc_godot_3_project_settings:
+
+Godot 3 Project Settings
+========================
+
++---------------+-------------------------+
+|  Version | v3.0 beta2                   |
```

Comment:
You need to *draw* the table, everything should be properly aligned in RST. It's a PITA but that's how it was designed... So the `|` need to be below the `+` at all times.